dev/com.ibm.ws.cdi.extension.bce_fat/bnd.bnd

```diff
@@ -0,0 +1,28 @@
+#*******************************************************************************
+# Copyright (c) 2018, 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+# Define the bundle for this FAT
+
+src: \
+    fat/src,\
+    
+tested.features:
```

Comment:
Since @jhanders34 mentioned Java 11, does this bnd also need
```
javac.source: 11
javac.target: 11

Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
```

Reply:
It might be allowed to have this part, though I'm not sure it's necessary.
```
javac.source: 11
javac.target: 11
```

I don't think it wants `Require-Capability` since it's not actually making a bundle.